OpenRA.Mods.Common/Traits/Power/CanPowerDown.cs

```diff
@@ -20,9 +20,13 @@ public class CanPowerDownInfo : UpgradableTraitInfo, Requires<PowerInfo>
 		public readonly bool CancelWhenDisabled = false;
 
 		public readonly string IndicatorImage = "poweroff";
-		public readonly string IndicatorSequence = "offline";
+		[SequenceReference("IndicatorImage")] public readonly string IndicatorSequence = "offline";
 
-		public readonly string IndicatorPalette = "chrome";
+		[PaletteReference] public readonly string IndicatorPalette = "chrome";
+
+		public readonly string NotificationType = "Sounds";
```

Comment:
This should be fixed in PlaceBeacon too.  Having a Notification that might actually be a Sound breaks our naming conventions, and limits the configuration options (mods might want to use both a notification and a sound effect).